src/DssSpell.sol

```diff
@@ -48,36 +152,321 @@ contract DssSpellAction is DssAction {
     //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
     // uint256 internal constant X_PCT_RATE = ;
+    uint256 internal constant SIX_PT_TWO_FIVE_PCT_RATE = 1000000001922394148741344865;
 
-    // ---------- Launch Project Funding ----------
-    address internal constant LAUNCH_PROJECT_FUNDING = 0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F;
+    // ---------- Math ----------
+    uint256 internal constant THOUSAND = 10**3;
+    uint256 internal constant MILLION  = 10**6;
+    uint256 internal constant WAD      = 10**18;
+    uint256 internal constant RAD      = 10**45;
 
-    // ---------- Contracts ----------
-    GemAbstract internal immutable MKR               = GemAbstract(DssExecLib.mkr());
+    // ---------- Phase 1b Addresses ----------
 
-    // ---------- Spark Proxy Spell ----------
-    // Spark Proxy: https://github.com/marsfoundation/sparklend-deployments/blob/bba4c57d54deb6a14490b897c12a949aa035a99b/script/output/1/primary-sce-latest.json#L2
-    address internal constant SPARK_PROXY = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
-    address internal constant SPARK_SPELL = 0x668C84584Ef8EeEd6BFb4FFB2a4Fa03231F8b241;
+    address internal constant USDS                         = 0xdC035D45d973E3EC169d2276DDab16f1e407384F;
```

Comment:
Those deployers are not found in the `addresses_deployers.sol`:
```
0x4Ec216c476175a236BD70026b984D4adECa0cfb8 // for USDS
0xD6ec7a1b1f4c42C5208fF68b2436Fab8CC593fB7 // for SUSDS
0x54eAde20f7DD1A67624626A3DB9408185eD0039e // for MCD_SPLIT
0x548DAc55f260AA4631F589Cb2fe72b5E9E4C93Dc // for MCD_VEST_SKY
0xEAB682cfE848FE2b42DA69a2591369EF589e8F27 // for REWARDS_USDS_01
0x4E65a603a9170fa572E276D1B70D6295D433bAc5 // for WRAPPER_USDS_LITE_PSM_USDC_A
```
It is [required by the checklist](https://github.com/makerdao/pe-checklists/blob/4b55876a9dec8b2c5ccaeb457fcbd134b629bb3a/spell/spell-reviewer-mainnet-checklist.md?plain=1#L94) to ensure `wards` are [properly tested](https://github.com/makerdao/spells-mainnet/blob/87487cde49a3797e22086f4557bd6ed4f9bc87f6/src/DssSpell.t.base.sol#L1752)